dojo/forms.py

```diff
@@ -433,6 +433,8 @@ class ImportScanForm(forms.Form):
     active = forms.BooleanField(help_text="Select if these findings are currently active.", required=False)
     verified = forms.BooleanField(help_text="Select if these findings have been verified.", required=False)
     scan_type = forms.ChoiceField(required=True, choices=SORTED_SCAN_TYPE_CHOICES)
+    environment = forms.ModelChoiceField(
+        queryset=Development_Environment.objects.all().order_by('name'))
```

Comment:
the import scan is broken in integration test, not sure, but could be caused by this?

Reply:
It is, I didn't know we had integration tests for the UI that's neat !